Sources/third_party/Experimental/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright © 2017 Xiaodi Wu (https://github.com/xwu).
```

Comment:
Please contact Xiaodi about this to see if he is willing to contribute it to the s4tf project.  We don't want to mix licenses in the swift-apis repos.

Reply:
Ya that sounds good I sent him an email about this! 😄